Cargo.toml

```diff
@@ -74,7 +74,7 @@ tiktoken-rs = { version = "0.5.9", optional = true }
 tokenizers = { version = "0.19.1", default_features = false, features = [
     "onig",
 ], optional = true }
-tree-sitter = { version = "0.22.6", optional = true }
+tree-sitter = { version = "0.20.2", optional = true }
```

Comment:
can we specify this with something like `>=0.20` so other ppl can lock the version to `0.22` if needed?

Reply:
I'm a little hesitant because of Cargo behavior around this https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-version-requirements

Unlike other package managers, Cargo doesn't necessarily do what you expect, and would still install the latest version of tree-sitter to use with text-splitter, not the version you specify. I ran into this issue with the tokenizers crate.